### ISSUE TITLE
No such file or directory: '/tmp/fmbench-read/tokenizer'

### ISSUE THREAD

**Issue by athewsey** [aws-samples/foundation-model-benchmarking-tool#102]
I've been trying to run FMBench with self-contained setup as described [on the README](https://github.com/aws-samples/foundation-model-benchmarking-tool?tab=readme-ov-file#run-fmbench-on-amazon-ec2-with-no-dependency-on-amazon-s3) to test whether Python 3.10 can be supported as per #94 

(But one caveat that I'm running on a SageMaker Notebook Instance rather than plain EC2)

Setting up conda & Poetry, then running `./copy_s3_content.sh` and `./debug.sh`, my debug script fails in notebook 0 with error:

```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/fmbench-read/tokenizer'
```

Sure enough, if I `ls /tmp/fmbench-read` I see folders for `configs`, `prompt_template`, `scripts`, and `source_data` - but no tokenizers... What's the expected way to set up the local tokenizers folder, and could we get it automated within `./copy_s3_content.sh`?

**Comment by athewsey** [comment]
It seems like copying the contents of [src/fmbench/tokenizer](https://github.com/aws-samples/foundation-model-benchmarking-tool/tree/main/src/fmbench/tokenizer) into the target `tmp` folder might be sufficient? But I'm not sure if the `copy_s3_content.sh` script is expected to be used in any contexts other than a git clone of this whole repo?

**Comment by aarora79** [comment]
Fixed as of v1.0.45.
